README.md

```diff
@@ -186,24 +207,20 @@ There are a few bugs in Zebra that we're still working on fixing:
 - [Interrupt handler does not work when a blocking task is running #1351](https://github.com/ZcashFoundation/zebra/issues/1351)
   - Zebra should eventually exit once the task finishes. Or you can forcibly terminate the process.
 
-Zebra's state commits changes using database transactions.
-If you forcibly terminate it, or it panics, any incomplete changes will be rolled back the next time it starts.
-
 ## Future Work
 
-In 2021, we intend to finish NU5 validation, start adding RPC support and start adding wallet integrations.
+In 2022, we intend to start adding RPC support and start adding wallet integrations.
 This phased approach allows us to test Zebra's independent implementation of the
 consensus rules, before asking users to entrust it with their funds.
 
 Features:
-- Full consensus rule validation
-- Wallet functionality
 - RPC functionality
+- Wallet functionality
 
 Performance and Reliability:
 - Reliable syncing on Testnet
 - Reliable syncing under poor network conditions
-- Batch verification
+- Additional batch verification
```

Comment:
@conradoplg @ftm1000 can you double-check my changes to the 2022 Zebra goals in the README?

Reply:
Seems good!